sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/logs/_exporter.py

```diff
@@ -80,8 +80,10 @@ def shutdown(self) -> None:
         if self.storage:
             self.storage.close()
 
-    def _log_to_envelope(self, log_data: LogData) -> TelemetryItem:
+    def _log_to_envelope(self, log_data: LogData) -> Union[TelemetryItem, None]:
         envelope = _convert_log_to_envelope(log_data)
+        if envelope is None:
```

Comment:
Should we purge empty logs prior to doing the work to convert them? Seems like this might save processing time if possible.